src/platform/packages/shared/kbn-unified-metrics-grid/src/components/trace_metrics_grid/index.tsx

```diff
@@ -0,0 +1,133 @@
+/*
```

Comment:
Is it possible to reuse the metrics_grid here? If we extract some of the fields' logic. We have a common structure already, and the charts here could be built using a single component passing different props instead of a component per chart ( so with some refactoring, I think we can merge those, or are there other concerns?)